.circleci/config.yml

```diff
@@ -97,6 +97,7 @@ version: 2
 jobs:
   lint:
     machine:
+      image: ubuntu-2004:202010-01
```

Comment:
Machine images are set to 20.04 through all machine-executor jobs in the CI config. :+1: